redeploy.sh

```diff
@@ -0,0 +1,28 @@
+set -x
```

Comment:
This might be a handy script for deploying this charm over and over -- but let's exclude from the PR for the time being